core/renderers/geras/path_object.js

```diff
@@ -24,7 +24,9 @@
 
 goog.provide('Blockly.geras.PathObject');
 
+goog.require('Blockly.blockRendering.ConstantProvider');
```

Comment:
We should use ``Blockly.geras.ConstantProvider`` only here (that in turn would require ``Blockly.blockRendering.ConstantProvider``). so there's no need for this.